src/l10n/ru.js

```diff
@@ -0,0 +1,354 @@
+            'Creating tag failed'                 : 'Не удалось создать тег',
+            'Edit tag'                            : 'Редактировать тег',
+            'Tag saved'                           : 'Тег сохранён',
+            'Saving tag failed'                   : 'Не удалось сохранить тег',
+            'Delete tag'                          : 'Удалить тег',
+            'Do you want to delete the tag'       : 'Точно удалить?',
+            'Tag deleted'                         : 'Тег удалён',
+            'Deleting tag failed'                 : 'Не удалось удалить тег',
+            'Tag restored'                        : 'Тег восстановлен',
+            'Restoring tag failed'                : 'Не удалось восстановить тег',
+            'Create password'                     : 'Создать пароль',
+            'Password created'                    : 'Пароль создан',
+            'Creating password failed'            : 'Не удалось создать пароль',
+            'Edit password'                       : 'Изменить пароль',
+            'Password saved'                      : 'Пароль сохранён',
+            'Saving password failed'              : 'Не удалось сохранить пароль',
+            'Password moved'                      : 'Пароль перемещён',
+            'Moving password failed'              : 'Не удалось переместить пароль',
+            'Delete password'                     : 'Удалить пароль',
+            'Do you want to delete the password'  : 'Точно удалить?',
+            'Password deleted'                    : 'Пароль удалён',
+            'Password restored'                   : 'Пароль восстановлен',
+            'Restoring password failed'           : 'Не удалось восстановить пароль',
+            'Open Url'                            : 'Открыть ссылку',
+            'Copy Url'                            : 'Скопировать ссылку',
+            'Copy User'                           : 'Скопировать имя',
+            'Copy Password'                       : 'Скопировать пароль',
+            '{element} was copied to clipboard'   : '{element} скопирован в буфер обмена',
+            'Last modified on {date}'             : 'Изменено {date}',
+            'Restore revision'                    : 'Восстановить',
+            'Revision restored'                   : 'Ревизия восстановлена',
+            'Restoring revision failed'           : 'Не удалось восстановить выбранную ревизию',
+            'Do you want to restore the revision?': 'Хотите восстановить данную ревизию?',
+            'Shared with me'                      : 'Поделились со мной',
+            'Shared by me'                        : 'Мною предоставлен доступ',
+            'Statistics'                          : 'Статистика',
+            'Created on'                          : 'Создано',
+            'Last updated'                        : 'Обновлено',
+            '{count} revisions'                   : 'Ревизий: {count}',
```

Comment:
This is being used in the "Details" view of a password. I think it looks better in the traditional way without the `:`

Reply:
I did it for the same reason as above. 
And in this case it looks better as I wrote.